include/LightGBM/feature_group.h

```diff
@@ -228,6 +228,14 @@ class FeatureGroup {
     return bin_data_->GetIterator(min_bin, max_bin, most_freq_bin);
   }
 
+  inline size_t FeatureGroupSizesInByte() {
+    return bin_data_->SizesInByte();
+  }
+
+  inline void* FeatureGroupData() {
+    return bin_data_->get_data();
+  }
```

Comment:
maybe throw excpetion when use `is_multi_val_` ? or check the `is_multi_val_` if it is not possible.